src/Workspaces/Core/Portable/SolutionCrawler/ServiceFeatureOnOffOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Shared.Options
+{
+    internal static class ServiceFeatureOnOffOptions
+    {
+        /// <summary>
+        /// This option is used by TypeScript.
+        /// </summary>
+        [Obsolete("Use the new option SolutionCrawlerOptions.BackgroundAnalysisScopeOption")]
```

Comment:
If you can put "TypeScript" in the Obsolete message I'd appreciate that, as I often grep for "Obsolete.*TypesScript" to see what stuff we can potentially delete.